porfile.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header>
+        <nav class="navbar">
+            <a href="index.html" class="brand">My Logo</a>
```

Comment:
- [ ] Your logo is not supposed to be inside the `nav` element. Kindly remove it and place it directly inside the `header`. Also, change the logo text from `my Logo` to `Your name`.